src/cloudant/design_document.py

```diff
@@ -522,6 +595,17 @@ def get_list_function(self, list_name):
         """
         return self.lists.get(list_name)
 
+    def get_show_function(self, show_name):
+        """
+        Retrieves a specific show function from the locally cached DesignDocument
+        shows dictionary by name.
+
+        :param str show_name: Name used to identify the show function.
+
+        :returns: Index dictionary for the specified show function name
```

Comment:
I'm a bit confused by `Index dictionary` in this returns statement. Doesn't it just return the string of the function?
I note that both `get_index`(self, index_name) and `get_list_function(self, list_name)` use the same terminology, but I think it only makes sense in the index case so probably need to fix this and the `list` case.